python/src/uagents/agent.py

```diff
@@ -255,6 +255,11 @@ def __init__(
                 self._port, self._loop, self._queries, logger=self._logger
             )
 
+        # define default error message handler
+        @self.on_message(ErrorMessage)
+        async def _handle_error_message(ctx: Context, sender: str, msg: ErrorMessage):
+            ctx.logger.warning(f"Received error message from {sender}: {msg.error}")
```

Comment:
Why not use `logger.error()`? It could be misleading when the text refers to an error message and the agent only logs a warning.

Reply:
Yes, agreed. I was thinking that errors coming back from another agent might not rise to the level of an internal agent exception, but it's more likely that an error like this does indicate a problem in the agent who's receiving the error message. Updated to `logger.exception`.